ecp5/family.cmake

```diff
@@ -6,18 +6,17 @@ if (NOT EXTERNAL_CHIPDB)
         set(TRELLIS_ROOT "/usr/local/share/trellis")
     endif()
 
-    file(GLOB found_pytrellis ${TRELLIS_ROOT}/libtrellis/pytrellis.*
-                              /usr/lib/pytrellis.*
-                              /usr/lib64/pytrellis.*
-                              /usr/lib/trellis/pytrellis.*
-                              /usr/lib64/trellis/pytrellis.*)
+    if (NOT DEFINED PYTRELLIS_LIBDIR)
+        find_library(PYTRELLIS pytrellis.so
+            PATH_SUFFIXES trellis
+            DOC "Location of pytrellis library")
 
-    if ("${found_pytrellis}" STREQUAL "")
-        message(FATAL_ERROR "failed to locate pytrellis library!")
-    endif()
+        if ("${PYTRELLIS}" STREQUAL "PYTRELLIS-NOTFOUND")
+            message(FATAL_ERROR "Failed to locate pytrellis library!")
+        endif()
 
-    list(GET found_pytrellis 0 PYTRELLIS_LIB)
-    get_filename_component(PYTRELLIS_LIBDIR ${PYTRELLIS_LIB} DIRECTORY)
+        get_filename_component(PYTRELLIS_LIBDIR ${PYTRELLIS} DIRECTORY)
+    endif()
```

Comment:
AFAICT this is just a much nicer and cleaner (and canonical, from a cmake kung-fu p.o.v.) way to find the pytrellis.so library path, and it keeps working fine for me when building a Fedora package.

Also, anything below this point seems to be whitespace indentation cleanup only.

Thanks for the heads-up!
`Reviewed-by: Gabriel Somlo <gsomlo@gmail.com>`